addon/-private/data-view/radar/radar.js

```diff
@@ -289,6 +289,21 @@ export default class Radar {
     }
   }
 
+  _updatePoolContents() {
```

Comment:
We can remove this function. `_updateVirtualComponents` will always update all of the items in the pool, we just need to make sure that `itemDelta` is zero if we don't want it to jump around (like it was in the dbmon example).